src/ui/A/A.jsx

```diff
@@ -58,13 +59,24 @@ function A({ to, hook, variant = 'default', children, isExternal, ...props }) {
     'inline-flex items-center gap-1'
   )
 
+  const handleClick = () => {
+    if (props.onClick) {
+      props.onClick()
+    }
+    if (to) {
+      metrics.increment(`link.click.${to.pageName}`)
+    } else {
+      metrics.increment(`link.click.${hook}`)
+    }
```

Comment:
Thoughts on refining this check to ensure that `pageName` exists or adding in a `.?`, as this isn't TypeSafe atm, just so we don't crash the App trying to collect metrics?

Reply:
I think the ? would be unnecessary here since we're already guarding on "to" being present on line 66, nor would it crash the app.

Here's an example:
![Screenshot 2024-06-13 at 10 40 03 AM](https://github.com/codecov/gazebo/assets/159853603/31f43e47-dbe6-459c-aa30-f2642d2be210)

Reply:
ah yea thought the type error would be one level down